app/components/instant-article-field/instant-article-field.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+angular.module('instantArticleField', ['bulbsCmsApp.settings', 'cms.instant_article.config', 'lodash'])
+  .directive('instantArticleField', ['routes', '_', 'InstantArticleConfig', function(routes, _, InstantArticleConfig) {
+    return {
+      templateUrl: routes.COMPONENTS_URL + 'instant-article-field/instant-article-field.html',
+      restrict: 'E',
+      scope: {
+        content: '='
+      },
+      link: function(scope, element, attrs) {
+        scope.instantArticleEnabled = false;
+
+        scope.setActive = function() {
+          scope.content.instant_article = true;
+        };
+
+        scope.setInactive = function() {
+          scope.content.instant_article = false;
+        };
+
+        scope.initInstantArticleEnabled = function() {
+          scope.instantArticleEnabled = _.includes(InstantArticleConfig.getSupportedFeatureTypes(), scope.content.feature_type);
+        };
+
+        scope.initInstantArticleEnabled();
```

Comment:
Use something like [this directive builder](https://github.com/theonion/cms-components/blob/master/resources/js/testing/test-helper.js#L3) to control when the directive is built, then test what's available on its `scope`.

The test currently testing this isn't as rigorous as it could be since it's calling the function after it's already been called once.